src/config.ts

```diff
@@ -268,21 +268,19 @@ export const CLIENT_OPTIONS: ClientOptions = {
 		'DIRECT_MESSAGE_REACTIONS'
 	],
 	loadDefaultErrorListeners: false,
-	makeCache: Options.cacheWithLimits({
-		...Options.defaultMakeCacheSettings,
-		MessageManager: {
-			sweepInterval: 180,
-			sweepFilter: LimitedCollection.filterByLifetime({
-				lifetime: 900,
-				getComparisonTimestamp: (m) => m.editedTimestamp ?? m.createdTimestamp
-			})
+	makeCache: {},
```

Comment:
Oh i get it now...lol
```suggestion
	makeCache: Options.cacheEverything(),
```